Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -271,6 +271,35 @@
       - state: inhand-right-blade
         shader: unshaded
 
+- type: entity
+  name: energy sword
```

Comment:
rename energy sword back to energy cutlass because we already have traitor energy sword and hos energy sword and we're starting to have so many weapons with the same name